genesyscloud/integration_credential/genesyscloud_integration_credential_init_test.go

```diff
@@ -31,6 +33,7 @@ func (r *registerTestInstance) registerTestResources() {
 	defer r.resourceMapMutex.Unlock()
 
 	providerResources["genesyscloud_integration_credential"] = ResourceIntegrationCredential()
+	providerResources["genesyscloud_oauth_client"] = oauth.ResourceOAuthClient()
```

Comment:
Adding a dependency for the genesys_oauth_client so we can test the situation where we are adding a Genesys Cloud OAuthClient